sci-astronomy/xephem/xephem-3.7.7-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs
+
+DESCRIPTION="Interactive tool for astronomical ephemeris and sky simulation"
+HOMEPAGE="https://www.clearskyinstitute.com/xephem"
+SRC_URI="https://www.clearskyinstitute.com/xephem/${P}.tgz"
+
+LICENSE="XEphem"
+SLOT=0
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND=">=x11-libs/motif-2.3:0
+	virtual/jpeg:0=
+	media-libs/libpng:0="
+DEPEND="${RDEPEND}
+	sys-apps/groff"
+
+HTML_DOCS=( GUI/xephem/help/. )
+DOCS=( README )
+
+PATCHES=(
+	"${FILESDIR}/${P}-respect_env_vars.patch"
+	"${FILESDIR}/${P}-implicits.patch"
+	"${FILESDIR}/${P}-no_xprint.patch"
+)
+
+src_prepare() {
+	default
+	echo > "${T}"/XEphem "XEphem.ShareDir: /usr/share/${PN}" || die
+	echo > "${T}"/99xephem "XEHELPURL=/usr/share/doc/${PF}/html/xephem.html" || die
+}
+
+src_compile() {
+	tc-export CC AR RANLIB
+	emake -C GUI/xephem
+}
+
+src_install() {
+	insinto /usr/share/X11/app-defaults
+	doins "${T}"/XEphem
+	doenvd "${T}"/99xephem
```

Comment:
consider ditching both these and doing the `newins -` `newenvd -` trick without resorting to temporaries